Navdeep-Singh-Ghotra/gitflow
# Into staging from Qa (#55)

* testautopr

* fixednames

* s

* v2

* peter-evans/create-pull-request@v3

* pr_branch to branch

* should work

* test

* indents

* indent again

* testit

* test2

* correcting token

* aha

* Update report

* reorderr

* Update report

* add staging

* milestone

* Update action.yml

* another change

* Update report

* Update report

* Update report

* Update report

* Testautopr (#47)

* changed branches

* changes for other branches

* added multiple files

* on pull_request

* commit all instead of one

* on pullrequestclosed on develop

* correcting sequence

* Testautopr (#48)

* changed branches

* changes for other branches

* added multiple files

* on pull_request

* commit all instead of one

* on pullrequestclosed on develop

* correcting sequence

* update develop (#49)

* Changes by create-pull-request action (#53)

* changed branches

* changes for other branches

* added multiple files

* on pull_request

* commit all instead of one

* on pullrequestclosed on develop

* update develop

* correcting sequence

* dev pr

* keep upto date

* resolove

Co-authored-by: Navdeep-Singh-Ghotra <NavdeepSingh.Ghotra@gmail.com>

* Into qa from Develop (#54)

* changed branches

* changes for other branches

* added multiple files

* on pull_request

* commit all instead of one

* on pullrequestclosed on develop

* update develop

* correcting sequence

* dev pr

* keep upto date

* resolove

Co-authored-by: Navdeep-Singh-Ghotra <Navdeep-Singh-Ghotra@users.noreply.github.com>
Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
